internal/pkg/admission/reconciler.go

```diff
@@ -175,7 +187,16 @@ func (r *Reconciler) Reconcile(
 		&policyServer.Status.Conditions,
 		policiesv1alpha2.PolicyServerDeploymentReconciled,
 	)
-	// TODO reconcile service and webhook
+
+	if err := r.reconcilePolicyServerService(ctx, policyServer); err != nil {
+		setFalseConditionType(
+			&policyServer.Status.Conditions,
+			policiesv1alpha2.PolicyServerServiceReconciled,
+			fmt.Sprintf("error reconciling service: %v", err),
+		)
+		return err
+	}
```

Comment:
We are missing the `SetTrueConditionType` that sets the service reconciliation condition to `true`.

Reply:
yes, I totally missed that. Will be added in the next commit. Thanks!